.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
OK, sounds good, thanks!

Please still reply "Done" etc. going forward so that we can keep track of the resolution of individual comments. I know we haven't really done it in the past, but this case shows that things can get missed.